src/App.js

```diff
@@ -14,11 +14,70 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    values: [],
```

Comment:
```suggestion
    value: null,
```
In fact, you have only one value selected. no need to create an array